src/proto/NEAR.proto

```diff
@@ -0,0 +1,92 @@
+syntax = "proto3";
+
+package TW.NEAR.Proto;
+option java_package = "wallet.core.jni.proto";
+
+message PublicKey {
+    uint32 key_type = 1;
+    bytes data = 2;
+}
+
+message FunctionCallPermission {
+    bytes allowance = 1; // uint128
+    string receiver_id = 2;
+}
+
+message FullAccessPermission {
+}
+
+message AccessKey {
+    uint64 nonce = 1;
+    oneof permission {
+        FunctionCallPermission function_call = 2;
+        FullAccessPermission full_access = 3;
+    }
+}
+
+message CreateAccount {
```

Comment:
this is also emtpy

Reply:
this is intentional, needed for future extension possibility